docs/troubleshooting.rst

```diff
@@ -193,3 +193,31 @@ Now build again::
     tutor images build
 
 All build commands should now make use of the newly configured builder. To later revert to the default builder, run ``docker buildx use default``. 
+
+Can't override styles using Indigo Theme for MFEs
+-------------------------------------------------
+
+The indigo theme can’t override styles for MFEs directly. It overrides the styles for edx-platform. In case of MFEs, `@edx/brand <https://github.com/openedx/brand-openedx>`_ is used to override the styles. Then, customize the ``@edx/brand`` package to your preferences and include this customized package in `tutor-indigo` plugin. In this way, styles can be overidden::
+
+
+    hooks.Filters.ENV_PATCHES.add_item((
+                "mfe-dockerfile-post-npm-install",
+                """
+    RUN npm install '@edx/brand@npm:custom-brand-package'
+    RUN npm install '@edx/brand@git+https://github.com/username/brand-openedx.git#custom-branch'
+    """,
+            ))
+
+
+NPM Dependency Conflict When overriding ``@edx/frontend-component-header`` or ``@edx/frontend-component-footer``
```

Comment:
Should we move this section to the tutor-mfe plugin? I think we should.

Reply:
It should at the very least _also_ be there.   Or, at least, a link from there to here.  (Or vice-versa.)